index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>CV of Aleksei Bulgak</title>
+  <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css" />
```

Comment:
not necessary to write type="text/css" according to http://codeguide.academy/html-css.html, because it's a default value for this attribute